### PR TITLE
Fix Sticky PageHeader

### PR DESCRIPTION
**Pull request by AmanShurafa**
Sticky Position css rule needed to be moved to the root div
- Moved inline styling to css file

### CODE REVIEW

File: src/PageHeader.js [hunk 1]
```diff
@@ -26,14 +26,12 @@ class PageHeader extends Component {
     }
   }
   render() {
-    if (this.state.stickyHeader) {
-      var stickyClass = {
-        top: '0.000em',
-      };
-    }
+    var headerContainerClass = this.state.stickyHeader
```
Comment (ratneshraval) [inline src/PageHeader.js]: Can we use const/let?

File: src/styles/pageheader.scss [hunk 2]
```diff
@@ -2,6 +2,12 @@
  PAGE HEADER STYLING
 ***/
 
+.page-header-container {
+  position: sticky;
```
Comment (brunoOSI) [inline src/styles/pageheader.scss]: did you try this on IE11 ? I'm sure this will need a polyfill.

File: src/PageHeader.js (hunk 1 above)
Comment (AmanShurafa) [reply src/PageHeader.js]: Done

File: src/styles/pageheader.scss (hunk 2 above)
Comment (AmanShurafa) [reply src/styles/pageheader.scss]: Yeah, this is not to add support for IE11. It's just to fix the existing functionality and make apply to page header and it's children.

Comment (brunoOSI) [reply src/styles/pageheader.scss]: everything we ship should work on IE11

**Comment by eckdanny-osi** [comment]
http://jenkins.cwds.tabordasolutions.net/jenkins/job/react-wood-duck/79/ is hung up. 

You can try pushing another commit to trigger a build, but i don't think anybody has perms to kill the zombie.

**Comment by AmanShurafa** [comment]
I forced pushed with a new sha. Nothing happened

**Comment by NagaPradeepGajula-osi** [comment]
Ran the test suite and installed the branch version in a application locally and the functionality is working fine.

**Pull request merged**